java-kubectl/README.md

```diff
@@ -0,0 +1,114 @@
+# Payroll
+
+This guides shows an example of developing a Spring Boot Java application with cnd. It's based on [Spring's gs-rest-service example](https://github.com/spring-guides/gs-rest-service).
+
+## Deploy the payroll service
```

Comment:
On the other examples we explained how to test it in local with minikube or Docker for Mac and what to expect. I understand it might not be necessary given the profile of user checking this example but I would at least add a list of requirements here specifying a "Kubernetes cluster", or something like that.